sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/options/DataLakePathScheduleDeletionOptions.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.file.datalake.options;
+
+import com.azure.core.annotation.Fluent;
+
+import java.time.Duration;
+import java.time.OffsetDateTime;
+
+/**
+ * Extended access options that may be passed to set when scheduling deletion for a path.
+ */
+@Fluent
+public class DataLakePathScheduleDeletionOptions {
+
+    private OffsetDateTime expiresOn;
+    private Duration timeToExpire;
```

Comment:
This should be a an int to pattern match the `BlobLeaseClient` - https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobLeaseClient.java#L129